### ISSUE TITLE
Special chars not escaped in the error __str__

### ISSUE THREAD

**Issue by UncleGoogle** [thombashi/pathvalidate#9]
```
In [14]: try:
    ...:     pathvalidate.validate_filepath('asdf\rsdf')
    ...: except Exception as e:
    ...:     print(e)
    ...: 
    ...: 
sdf', reason=INVALID_CHARACTERars=(\
```

**Comment by thombashi** [comment]
Thank you for your report.

The error fixed at `pathvalidate 0.28.2` as the following:

```pycon
>>> pathvalidate.__version__
'0.28.2'
>>> try:
...     pathvalidate.validate_filepath('asdf\rsdf')
... except Exception as e:
...     print(e)
...
invalid char found: invalid-chars=('\r'), value=''asdf\rsdf'', reason=INVALID_CHARACTER
```

Please try to upgrade the package.

**Comment by UncleGoogle** [comment]
Thanks for instant fix! repr for greater good.
